.github/workflows/github-actions.yml

```diff
@@ -88,7 +88,17 @@ jobs:
       #   run: |
       #     tree ${{ matrix.github_actions_runner_root }}
       - name: Execute Maven Install Target And OpenJCEPlus Provider Tests
-        run: mvn --batch-mode '-Dock.library.path=${{ github.workspace }}/OCK/' -Dtest=ibm.jceplus.junit.openjceplus.TestAll,ibm.jceplus.junit.openjceplus.integration.TestAll install
+        run: > 
```

Comment:
Is this required for multiple lines?

Reply:
Yes i saw this online as the syntax for specifying multiple lines in yaml files for a key value pair.